data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,405 @@
+			label initiative
+			`	"You lack the tools for such a mission. To my knowledge, no vessels capable of landing on gas giants have ever been developed by your kind. And you are certainly not able to make your intentions known to them in any meaningful sense."`
+			`	"But for the sake of the Incipias, I will assist you on this last point. If you cannot be dissuaded from your course of action, then it is best you are supplied a means of communication, so that your interactions may be on amicable terms. This one can produce you with a single one of our translators, which will allow you to relate your intentions to the species in their home system. Request that they sell you one of their ships, so that you are made capable to land on their tumultuous planets."`
+			`	It pauses before taking on a more serious tone. "However, you are forbidden from granting the young species any advanced technology of yours. If you concur with us on that point, we shall furnish you with a translator."`
+			label choice
+			choice
+				`	"If that's how I can help them, sure."`
+					goto yes
+				`	"Why do you not want me to sell my tech?"`
+			`	"Halting the natural increase of a species is unpredictable at the best of times. Your species lacks the wisdom to understand this, as you are young, and have not seen what we have seen." The Quarg pauses for a moment, lost in thought. "The foreseeable risks of even letting you contact them are monumental, and not something you should consider trivial."`
+				goto choice
+			label yes
+			`	"Happy I am that you agree to that condition. Bear in mind, Captain, that the slightest misstep may have grave consequences." The Quarg stares you down. "We will be watching your actions, to verify that they are peaceful and without ill or unwise intent. Please for a moment remain here," it says, and walks away.`
+			`	The Quarg only takes a moment to come back; it is now carrying a sleek silver box with a display on one side.`
+			`	"This is our translator, our link to the Incipias. It is able to translate their unwonted language into your speech as well. Be aware of the honor of me gifting this to you," it says.`
+			`	As you stow it away, the Quarg speaks once more. "To satisfactorily utilize this device, you must direct it towards the source of the communication, so that it is made able to discern the relevant emissions. Given the complexity of their language, the translator might not be able to adequately translate some aspects of their obscure speech."`
+			`	It then looks you directly in the eyes. "Remember what I have told you, human. The stars have no place for those who would act foolishly and thoughtlessly towards their fellow beings." The Quarg then walks away, leaving you alone.`
+				accept
+			label declined
+			`	"We are gracious for your understanding, but remember thus: the Incipias are under our watch and care. None shall come to harm them without incurring our wrath." With these words the Quarg turns away.`
```

Comment:
```suggestion
			`	"We are gracious for your understanding, but remember thus: the Incipias are under our watch and care. None shall come to harm them without incurring our wrath." With these words the Quarg turns away.`
```
Just commenting that I feel this is a better place for them to be a bit more on the nose about intervening, and since I suggested prior that the Archon says intervene instead of 'wipe you out', it makes sense here that the Quarg are a bit more direct.